### DISCUSSION TITLE
Can any GitHub Insider help me

### DISCUSSION THREAD

**Discussion by darrenwjlau** [community/community#69730] [GitHub Education] [answered]
I submitted two time and the rejected reason was not using school email.

The problem is
I am using my school email by linking it to my personal and selected it during sending the request

First request I used a student id
Second request I used a government enrollment letter
Thrid request (not yet approved or denied) my school transcript

My personal email is <REMOVED>

My school email is <REMOVED>

<personal info removed>

**Comment by mantasu** [comment]
The only option is to choose an email that has `ed` or `edu` in it, so personal email won't do. You must get a verification code in your student email inbox.

**Comment by ayangabryl** [comment]
Make your school email primary email or remove the personal email. Just get it back once approve. 
Note: Do not post your documents here, Thank you!

**Comment by lasr21** [comment, marked as answer]
solved :)

**Comment by gitstua** [comment]
I've removed your private information from [this post](https://github.com/orgs/community/discussions/69730) as posting identifiable information is not allowed, per our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct).

Glad to hear you have resolved this.

**Reply by killdrill** [reply]
can you get some info about remote service from my own machintosh /OWC

**Comment by killdrill** [comment]
i need personal information from selective firms that can be remote active while doctor meeting leads to correct signing for correct medicines.
Try Microsoft for personal info and remoting by OWC. Authersized by Apple.NET ( Gecho echo family Co. Trade Contractor Steve Job\next leader of Microsoft.NET... Mark 16/Cloudflare.com erath security healt agreement.
